.vscode/settings.json

```diff
@@ -168,6 +168,7 @@
 
 	// Temporarily enabled for self-hosting
 	"terminal.integrated.enableStickyScroll": true,
```

Comment:
I think the line above can be deleted?

Reply:
I wanted to keep it for a few days for the transition